UnityProject/Assets/Scripts/Health/Objects/Integrity.cs

```diff
@@ -55,9 +55,9 @@ public class Integrity : NetworkBehaviour, IHealth, IFireExposable, IRightClicka
 	public Resistances Resistances = new Resistances();
 
 	/// <summary>
-	/// Below this temperature, the object will take no damage from fire or heat and won't ignite.
+	/// Below this temperature (in degrees Celsius) the object will take no damage from fire or heat and won't ignite.
```

Comment:
It's actually Kelvin. All temps in the game are kelvin unless otherwise stated. The reason it was having no effect (despite this being extremely low) was that this field currently only gets used when an object is in an exposed hot spot (where there's a visible burning sprite on the tile). Even if the heat on a tile is really high the object doesn't currently take any damage. We haven't yet implemented temperature exposure, only fire exposure.

There is an equivalent field in tg used by a few objects such as canisters, temperature_resistance, and a "temperature_expose" method.

For now let's just change this comment to say Kelvin. The default of 100 was just a random guess, so when we implement temperature exposure I will most likely refactor to make it more sane.

Reply:
edit: also remembered the reason for this field was so I could give canisters a higher value so they wouldn't all pop like balloons when they are in a fire

Reply:
IMO this field should actually be removed so we match TG logic, but keep it in for now. I'll make a ticket.